lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.html

```diff
@@ -0,0 +1,85 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+{% load i18n %}
+{% load django_markup %}
+{% load static %}
+{% block content %}
+{# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
```

Comment:
all this styling is somewhat janky - though i was following Mike Terry's suggestion of following the standard of the other ace emails
though not sure what a good improvement would be, besides modifying ace to use a tool other than django templates which would allow writing css in stylesheets and then inlining it during a static build process

Reply:
yes inline styling seems to be the convention for the email templates, there's just more of it needed for this email than most.  About email clients where media queries don't work, maybe we can find such a client to test with as part of QA'ing this

Reply:
i suppose improving this architecture wouldn't belong in this PR anyways, but this might be a potential area for investment